lib/models/subscription.js

```diff
@@ -133,6 +139,17 @@ module.exports = (sequelize, DataTypes) => {
       this.changed('settings', true);
     },
 
+    async updateGithubName(githubName) {
+      if (this.githubName !== githubName) {
+        await this.update({ githubName });
+      }
+    },
```

Comment:
💡 I think it might be good to protect against accidentally updating to `null` here. if the passed in `githubName` is `null` (passed by accident from the calling function), then we should probably abort and log a warning?

Reply:
I added a conditional. I don't think we have to abort because if some webhook changes or a new webhook event has a different _shape_ we might be aborting the deletion for not a good reason.